StepDefinitions/FacebookSD/SignupSD.js

```diff
@@ -0,0 +1,93 @@
+const { Given, Then } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const moment = require("moment");
+const Homepage = require("../../Pages/Facebook/Homepage");
+const SignupPage = require("../../Pages/Facebook/SignupPage");
+
+Given(/^I am on facebook homepage$/, async function () {
+    await browser.url('/');
+    await browser.pause(5000);
+})
+
+Given(/^I click on Create New Account button$/, async function () {
+    const homepage = new Homepage();
+    const signuppage = new SignupPage();
```

Comment:
you can bring above two line outside function to make it global and use the same object inside all other functions.